tests/e2e/test_e2e.py

```diff
@@ -240,19 +239,44 @@ def test_prompt_manager_no_model_params():
         )
 
 
-def test_prompt_manager_no_model_params_undeployed():
+def test_prompt_manager_undeployed_latest_revision():
     mgr = UsedByCiDontDeleteNoParamsUndeployedPromptManager(
```

Comment:
still need to update the autogeneration scripts to not create a separate "Undeployed" class but will do that separately